client/src/redux/reducers/datafiles.reducers.js

```diff
@@ -407,6 +407,14 @@ export function files(state = initialFilesState, action) {
           },
         },
       };
+    case 'PROJECTS_CLEAR_FILE_SELECTION':
```

Comment:
Perhaps this could be named more generally, since it is not necessarily dependent on projects
```suggestion
    case 'DATA_FILES_CLEAR_FILE_SELECTION':
```